src/usdb_syncer/notes_parser.py

```diff
@@ -517,6 +523,18 @@ def fix_touching_notes(self) -> None:
                     if line.end() == self.notes.track_1[num_line + 1].start:
                         line.notes[-1].shorten()
 
+    def fix_pitch_values(self) -> None:
+        min_pitch = 1000
+        for note in self.notes.all_notes():
+            if min_pitch is None or note.pitch < min_pitch:
+                min_pitch = note.pitch
```

Comment:
I believe `min_pitch = min(note.pitch for note in self.notes.all_notes())` should also work.